includes/checksum.inc

```diff
@@ -8,6 +8,8 @@
 /**
  * Modifies an existing object's datastream checksums.
  *
+ * Used in the retroactive checksum generation batch.
+ *
```

Comment:
Nice catch @bradspry.  I'll add to https://github.com/Islandora/islandora_checksum/pull/44 later today.